hoomd_organics/base/simulation.py

```diff
@@ -395,20 +415,33 @@ def run_update_volume(
             The temperature to use during shrinking.
         tau_kt : float; required
             Thermostat coupling period (in simulation time units)
-        final_box_lengths : np.ndarray, shape=(3,), dtype=float; required
+        final_box_lengths : np.ndarray, shape=(3,), dtype=float; optional
             The final box edge lengths in (x, y, z) order
+        final_density : float; optional
+            The final density of the simulation
 
         """
+        if final_box_lengths is None and final_density is None:
```

Comment:
We probably need a check that throws an error if both `final_box_lengths` and `final_density` are given.